src/java.base/linux/classes/jdk/internal/platform/CgroupSubsystemFactory.java

```diff
@@ -121,6 +91,11 @@ public static CgroupMetrics create(Optional<CgroupTypeResult> optResult) {
         }
     }
 
+    private static void warn(String msg) {
+        Logger logger = System.getLogger("jdk.internal.platform");
+        logger.log(Level.DEBUG, msg);
+    }
```

Comment:
This seems fine and uncontroversial. Suggested name change `log` over `warn`. Perhaps apply this as a separate change?